data/json/items/tool/lighting.json

```diff
@@ -637,15 +637,15 @@
     "id": "heavy_flashlight",
     "type": "TOOL",
     "name": { "str": "heavy-duty flashlight (off)", "str_pl": "heavy-duty flashlights (off)" },
-    "description": "A heavy-duty aluminum LED flashlight, often used by security guards.  It even works underwater, and it's not bad for smacking hooligans.  It also has a shoulder strap on it's handle, for easier transportation.",
-    "//": "https://www.amazon.com/dp/B00Y8AHZYC, todo add low brightness mode",
-    "//2": "volume is average between cylinder volume (without handle) and entire unit as rectangular. Should be fine since handle is big and hollow, and it can be strapped anyway.",
-    "weight": "720 g",
-    "volume": "4267 ml",
-    "longest_side": "25 cm",
-    "price": "46 USD",
-    "price_postapoc": "15 cent",
-    "material": [ "aluminum" ],
+    "description": "A heavy-duty LED flashlight with a durable polymer housing.  It's impact resistant, chemical resistant, waterproof and certified for use in hazardous environments.  Designed for firefighters, rescue personnel and miners but it can also be used for smacking hooligans.  It has a spring-loaded clip that can be attached to your torso.",
```

Comment:
```suggestion
    "description": "A heavy-duty LED flashlight with a durable polymer housing.  It's impact-resistant, chemical-resistant, waterproof and certified for use in hazardous environments.  Designed for firefighters, rescue personnel and miners but it can also be used for smacking hooligans.  It has a spring-loaded clip that can be attached to your torso.",
```